package.json

```diff
@@ -158,6 +158,7 @@
     "recursive-readdir": "^2.2.2",
     "regenerator-runtime": "0.13.3",
     "rimraf": "^2.6.3",
+    "sharp": "^0.29.3",
```

Comment:
are we actually using this anywhere? the only references I see are here and in the lockfile

Reply:
Yeah Parcel uses it to resize the images. 

```js
import heroNarrow from 'url:../assets/internationalized_mobile@2x.png?width=976';
```

Alternative would be to resize in advance and commit all the versions I guess.

Reply:
i'm fine either way, parcel just auto picks it up if it's in the project deps though?

Reply:
yeah it actually auto installed it here